### ISSUE TITLE
Warnings and Test failure with Julia 1.10

### ISSUE THREAD

**Issue by ViralBShah** [JuliaMath/Polynomials.jl#556]
```
Precompiling project...
  31 dependencies successfully precompiled in 14 seconds. 20 already precompiled.
     Testing Running tests...
┌ Warning: ignoring the axis offset of the coefficient vector
└ @ Polynomials ~/.julia/packages/Polynomials/5ZhzG/src/polynomial-container-types/immutable-dense-polynomial.jl:60
┌ Warning: ignoring the axis offset of the coefficient vector
└ @ Polynomials ~/.julia/packages/Polynomials/5ZhzG/src/polynomial-container-types/immutable-dense-polynomial.jl:60
┌ Warning: ignoring the axis offset of the coefficient vector
└ @ Polynomials ~/.julia/packages/Polynomials/5ZhzG/src/polynomial-container-types/mutable-dense-polynomial.jl:9
┌ Warning: ignoring the axis offset of the coefficient vector
└ @ Polynomials ~/.julia/packages/Polynomials/5ZhzG/src/polynomial-container-types/mutable-dense-polynomial.jl:9
┌ Warning: ignoring the axis offset of the coefficient vector
└ @ Polynomials ~/.julia/packages/Polynomials/5ZhzG/src/polynomial-container-types/mutable-sparse-polynomial.jl:34
┌ Warning: ignoring the axis offset of the coefficient vector
└ @ Polynomials ~/.julia/packages/Polynomials/5ZhzG/src/polynomial-container-types/mutable-sparse-polynomial.jl:34
┌ Warning: Using the axis offset of the coefficient vector
└ @ Polynomials ~/.julia/packages/Polynomials/5ZhzG/src/polynomial-container-types/mutable-dense-laurent-polynomial.jl:19
┌ Warning: ignoring the axis offset of the coefficient vector
└ @ Polynomials ~/.julia/packages/Polynomials/5ZhzG/src/polynomial-container-types/mutable-dense-polynomial.jl:9
┌ Warning: ignoring the axis offset of the coefficient vector
└ @ Polynomials ~/.julia/packages/Polynomials/5ZhzG/src/polynomial-container-types/mutable-dense-polynomial.jl:9
┌ Warning: ignoring the axis offset of the coefficient vector
└ @ Polynomials ~/.julia/packages/Polynomials/5ZhzG/src/polynomial-container-types/mutable-sparse-vector-polynomial.jl:45
┌ Warning: ignoring the axis offset of the coefficient vector
└ @ Polynomials ~/.julia/packages/Polynomials/5ZhzG/src/polynomial-container-types/mutable-sparse-vector-polynomial.jl:45
P = FactoredPolynomial: Test Failed at /Users/viral/.julia/packages/Polynomials/5ZhzG/test/StandardBasis.jl:1357
  Expression: [p1[idx] for idx = eachindex(p1)] ==ᵗᶻ [1, 2, 0, 3]
   Evaluated: ComplexF64[1.0 - 1.6638121993696656e-17im, 2.0 - 1.4076796700415454e-16im, -8.326672684688674e-17 - 3.3306690738754696e-16im, 3.0 + 0.0im] ==ᵗᶻ [1, 2, 0, 3]

Stacktrace:
 [1] macro expansion
   @ ~/.julia/juliaup/julia-1.10.0+0.aarch64.apple.darwin14/share/julia/stdlib/v1.10/Test/src/Test.jl:672 [inlined]
 [2] macro expansion
   @ ~/.julia/packages/Polynomials/5ZhzG/test/StandardBasis.jl:1357 [inlined]
 [3] macro expansion
   @ ~/.julia/juliaup/julia-1.10.0+0.aarch64.apple.darwin14/share/julia/stdlib/v1.10/Test/src/Test.jl:1669 [inlined]
 [4] macro expansion
   @ ~/.julia/packages/Polynomials/5ZhzG/test/StandardBasis.jl:1306 [inlined]
 [5] macro expansion
   @ ~/.julia/juliaup/julia-1.10.0+0.aarch64.apple.darwin14/share/julia/stdlib/v1.10/Test/src/Test.jl:1577 [inlined]
 [6] top-level scope
   @ ~/.julia/packages/Polynomials/5ZhzG/test/StandardBasis.jl:1306
```

**Comment by jverzani** [comment]
Thanks for letting me know!!

**Comment by jverzani** [comment]
closed with #559
